Jenkinsfile_k8s

```diff
@@ -90,6 +90,12 @@ pipeline {
       }
         stages {
           stage('Prepare Environment'){
+            when {
+              anyOf {
+                  changeset "clusters/${K8S_CLUSTER}.yaml"
+                  changeset "config/**"
```

Comment:
Wondering if I shouldn't add "Jenkinsfile_k8s" to these paths (?)
Can be added later if needed.